pr_review_queue.py

```diff
@@ -244,24 +245,65 @@ def create_pr_review_queue(pull_request_list):
     Return a filtered list of pull requests according to these criteria:
         1. CI is green
         2. Not a draft
-        3. No changes requested
-        4. No merge conflicts
+    The resulting pull requests are grouped into four sections:
+        1. ‘Needs reviewer’: ping author about finding a reviewer
+        2. ‘Needs changes’: ping author to do the changes
+        3. ‘Needs review’: ping assigned reviewer/s
+        4. ‘Needs conflict resolution’: ping author to rebase/fix conflicts
     """
-    pr_review_queue = []
-    i = 0
+    needs_reviewer = []
+    needs_changes = []
+    needs_review = []
+    needs_conflict_resolution = []
+
     for pull_request in pull_request_list:
+        # 1. Needs reviewer
         if (pull_request["status"] == "success" and
             pull_request["draft"] == False and
             pull_request["changes_requested"] == False and
+            pull_request["mergeable_state"] != "dirty" and
+            pull_request["requested_reviewers"] == []):
+            entry = (f"*{pull_request['repo']}*:"
+                     f" <{pull_request['html_url']}|{pull_request['title']}>"
+                     f" (+{pull_request['additions']}/-{pull_request['deletions']})"
+                     f" needs a reviewer!>")
```

Comment:
```suggestion
                     f" needs a reviewer!")
```
the ">" character is just a copy and paste artefact I guess - and if this is just below the "These need a reviewer" queue we could as well skip it 
```suggestion
                     )
```